roles/activemq/tasks/main.yml

```diff
@@ -107,7 +107,8 @@
           line: admins={{ activemq_username }}
         - file: jetty-realm
           line: "{{ activemq_username }}: MD5:{{ activemq_password | md5 }}, admin"
-      no_log: true
```

Comment:
I know `no_log` is painful but without it `label` on iterator would not prevent from displaying passwords in Ansible output when ran in debug mode for instance